jsapp/scss/libs/_mdl.scss

```diff
@@ -1,7 +1,23 @@
 @import "../colors";
-
 // Theming variables, mixins and includes
 
+$black: #414453;
+$white: #fff;
+$cool-charcoal: lighten($black, 10%); // #575b70
+$cool-darkgray: lighten($black, 20%); // #6e738c
+$cool-gray: lighten($black, 30%); // #8a8ea3
+$cool-silver: lighten($black, 40%); // #a6aaba
+$cool-lightgray: lighten($black, 60%); // #e0e1e6
+$divider-color: lighten($black, 63%); // #e8e9ed
+$cool-whitegray: lighten($black, 67%); // #f4f4f6
+$c-desert-storm: #f8f8f8;
+$cool-blue: #2095f3;
+$cool-green: #52c5d0;
+$warm-green: #52ce7b;
+$cool-red: #ff556a;
+$cool-orange: #ffaa56;
+$cool-light-red: #d08b8c;
+
```

Comment:
This is probably some merge gone wrong, but these colors should be deleted as only `colors.scss` are valid ones :warning: